crates/stackable-operator/CHANGELOG.md

```diff
@@ -39,6 +40,7 @@ All notable changes to this project will be documented in this file.
 [#840]: https://github.com/stackabletech/operator-rs/pull/840
 [#841]: https://github.com/stackabletech/operator-rs/pull/841
 [#843]: https://github.com/stackabletech/operator-rs/pull/843
+[#853]: https://github.com/stackabletech/operator-rs/pull/853
```

Comment:
This line must be moved to the Unreleased section.

Reply:
Thanks for spotting that